### ISSUE TITLE
Should mention Account SID must be Live

### ISSUE THREAD

**Issue by kumasento** [TwilioDevEd/video-quickstart-node#6]
Hi, 

I tried with the **Test **Account SID, but there was always a Gateway error. Then I shifted to **Live **and everything works like a charm. 

I think it might be better to mention this in the README.

Thanks

**Comment by trnelson** [comment]
Agreed. Please make this update to the README file! Otherwise you get a 403 error, 31201 status code (generic error) and an output of "Could not connect to Twilio, undefined..."
